hyperspy/misc/test_utils.py

```diff
@@ -163,6 +168,52 @@ def assert_warns(message=None, category=None):
             raise ValueError(msg)
 
 
+def get_matplotlib_version_label():
+    """ Return a string describing the matplotlib version installed. The string
+    is used to label the references images for plot testing. """
+    version = LooseVersion(matplotlib.__version__)
+    if version >= LooseVersion('2.0.0'):
+        return 'mpl2'
+    else:
+        return 'mpl1'
+
+
+def reset_rcParams_default():
+    import matplotlib.pyplot as plt
+    plt.rcParams.clear()
+    plt.rcParams.update(plt.rcParamsDefault)
+
+
+@simple_decorator
+def switch_backend_mpl(function):
```

Comment:
I've just discovered that it's possible to set the mpl backend using [environement variables](https://github.com/matplotlib/matplotlib/blob/master/.travis.yml#L149). Isn't that preferable to this decorator?